tests/types.test.js

```diff
@@ -9,49 +9,92 @@ describe('Cases from app.js', () => {
   describe('apples', () => {
     it('should be an array', () => {
       const apples = cases.apples;
-      expect(apples).to.be.an('object'); // Begin here something is wrong, fix it.
+      expect(apples).to.be.an('array'); // Begin here something is wrong, fix it.
+    });
+    describe('apples', () => {
+      it('the length should be 4', () => {
+        const apples = cases.apples;
+        expect(apples.length).to.equal(4);
+      });
     });
     // write a test to check the correct length of the apples array.
+    describe('apples', () => {
+      it('should contain Red Delicious', () => {
+        const apples = cases.apples;
+        expect(apples).includes('Red Delicious', true);
+      })
+    });
     // write a test to check to see if the apples array contains 'Red Delicious'.
   });
 
   describe('Ben', () => {
     it('should be an object', () => {
       const Ben = cases.Ben;
-      expect(Ben).to.be.an('array');
+      expect(Ben).to.be.an('object');
     });
     // write a test to check if Ben's favorite band is 'Beastie Boys'.
+    describe('Ben', () => {
```

Comment:
Same here, no need to repeat `describe('Ben')` when the initial `describe()` wraps this statement.